CHANGELOG.md

```diff
@@ -67,9 +78,9 @@ See the [release announcement](TBD) for details.
 
 - [Support for pause-aware picking](https://github.com/godotengine/godot/pull/39421).
   - This breaks compatibility, but is not enabled by default for existing projects. See the project setting `physics/common/enable_pause_aware_picking`. It will be enabled by default for new projects created with 3.3.
-- [Cylinder collision shape support in GodotPhysics](https://github.com/godotengine/godot/pull/45854).
 - [CollisionObject can now display collision shape meshes](https://github.com/godotengine/godot/pull/45783).
-- [CollisionPolygon `margin` property for Bullet](https://github.com/godotengine/godot/pull/45855).
+- **Bullet:** [CollisionPolygon `margin` property](https://github.com/godotengine/godot/pull/45855).
+- **GodotPhysics:** [Cylinder collision shape support](https://github.com/godotengine/godot/pull/45854).
```

Comment:
Could we mark cylinder support as experimental in the changelog? It generally works, but there are already some known issues around kinematic bodies and things should get more stable for 3.4.